exporter/clickhousetracesexporter/writer.go

```diff
@@ -199,7 +200,19 @@ func (w *SpanWriter) writeIndexBatch(batchSpans []*Span) error {
 		}
 	}
 
-	return statement.Send()
+	start := time.Now()
+
+	err = statement.Send()
+
+	ctx, _ = tag.New(ctx,
+		tag.Upsert(exporterKey, string(component.DataTypeMetrics)),
```

Comment:
This should be traces
```suggestion
		tag.Upsert(exporterKey, string(component.DataTypeTraces)),
```